bluehost-wordpress-plugin.php

```diff
@@ -77,7 +101,7 @@ function () {
 					'id'           => 'bluehost',
 					'file'         => BLUEHOST_PLUGIN_FILE,
 					'brand'        => get_option( 'mm_brand', 'bluehost' ),
-					'platform'     => get_option( 'bh_platform', 'legacy' ),
+					'platform'     => bh_is_jarvis() ? 'jarvis' : 'legacy',
```

Comment:
Putting this here means it is doing the check on every single pageload, front-end or backend.

Reply:
Is this needed @wpalani ? I don't see container()->plugin()->platform used anywhere, I haven't checked modules though.